pebblo/app/config/service.py

```diff
@@ -1,17 +1,33 @@
 from contextlib import redirect_stderr, redirect_stdout
-from fastapi import FastAPI
 from io import StringIO
-import uvicorn
+from typing import Any
+from pathlib import Path
 import asyncio
-
-from pebblo.app.routers.local_ui_routers import local_ui_router_instance
+from fastapi import FastAPI, Response
 from fastapi.staticfiles import StaticFiles
-from pathlib import Path
+import uvicorn
+from pebblo.app.routers.local_ui_routers import local_ui_router_instance
+
 
 with redirect_stdout(StringIO()), redirect_stderr(StringIO()):
     from pebblo.app.routers.routers import router_instance
 
 
+class NoCacheStaticFiles(StaticFiles):
```

Comment:
Here we are overriding `StaticFiles`class and setting headers such that it will not use cached content, but it will always fetch latest data from server.